apps/scan/backend/src/app.ts

```diff
@@ -65,6 +66,8 @@ import {
 } from './util/diagnostics';
 import { saveReadinessReport } from './printing/readiness_report';
 
+const BALLOT_AUDIT_ID_KEY_LENGTH = 16;
```

Comment:
This is the secret key, which should be 256 bits, so 32 bytes.

Reply:
Switched to the EBC cipher per other comment, which made this point moot, as it only requires a key, not an IV/salt, so there's no value from using the password method. We also realized that it's nice to not have to rely on openssl's password->key derivation so that anyone can decrypt audit IDs without necessarily using openssl.